chrono_common/chronolog_types.h

```diff
@@ -54,6 +54,21 @@ class LogEvent
     {
         serT(storyId, eventTime, clientId, eventIndex, logRecord);
     }
+
+    bool operator==(const LogEvent &other) const
+    {
+        return (storyId == other.storyId && eventTime == other.eventTime && clientId == other.clientId &&
```

Comment:
I don't think (logRecord==other.logRecord) is needed. comparison of keys should suffice

Reply:
> I don't think (logRecord==other.logRecord) is needed. comparison of keys should suffice

Rolled back in the latest commit.